RedHatInsights/insights-inventory-frontend
# Add "No groups available" message to the filter

Implements https://issues.redhat.com/browse/RHINENG-1628.

This PR adds some new tests to the useGroupFilter hook and also makes the filter render "No groups available" text when there are no groups available for user.

## How to test

1. Make sure you have no groups available (created) for your account
2. Navigate to /inventory and check that the new message is displayed

## Screenshots

<img width="1106" alt="Screenshot 2023-11-20 at 16 56 44" src="https://github.com/RedHatInsights/insights-inventory-frontend/assets/31385370/d0218f47-db4d-405a-87aa-4817db520e4d">
